m0t1fs/linux_kernel/dir.c

```diff
@@ -334,6 +335,14 @@ int m0t1fs_setxattr(struct dentry *dentry, const char *name,
 
 	M0_THREAD_ENTER;
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,9,0)
+	/*
+	 * XXX: this 'dirty hack' is for oostore objects whose xattr name
```

Comment:
We don't allow the word "dirty" or "hack".
Let's remove it or find another phrase.

Reply:
Fixed.